CMakeLists.txt

```diff
@@ -238,7 +238,9 @@ if(WIN32 AND MSVC)
   # Replace the /Zi option (i.e. enable debugging information), if present, with
   # the /Z7 option (i.e. enable old-style debugging information) otherwise
   # caching (be it with buildcache or clcache) won't work.
-  replace_compiler_flag("/Z[iI]" "/Z7")
+  if(BUILDCACHE_EXE OR CCACHE_EXE)
```

Comment:
`CLCACHE` I suspect.

Reply:
Damn it! I guess that's why we have a reviewing process... 🙄 Done in commit c89cd35.